web/src/components/molecules/TopPage/TopPageContents.tsx

```diff
@@ -63,4 +101,11 @@ const Wrapper = styled.div`
   background: transparent;
 `;
 
+const TabsWrapper = styled.div`
+  background: #070707;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 48px;
+`;
```

Comment:
Can you double check if this component is being used as an extension in ReEarth Viz? If so, adding background color here will break the UI on the extension side (where we have theming)

Reply:
Thanks for the catch. Going to conditionally render these tabs to not render if the marketplace is used as an extension.